build/windows/installer/conf/fluent-bit-geneva.conf

```diff
@@ -30,6 +30,19 @@
     Record Computer ${HOSTNAME}
     Record AzureResourceId ${AKS_RESOURCE_ID}
 
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
+#${KubernetesFilterEnabled}    Merge_Log           Off
+#${KubernetesFilterEnabled}    Buffer_Size         0
+#${KubernetesFilterEnabled}    Use_Kubelet         true
+#${KubernetesFilterEnabled}    Kubelet_Port        10250
+#${KubernetesFilterEnabled}    Kubelet_Host        ${NODE_IP}
+#${KubernetesFilterEnabled}    tls.verify          Off
+#${KubernetesFilterEnabled}#${AnnotationBasedLogFilteringEnabled}    K8S-Logging.Exclude On
+#${KubernetesFilterEnabled}    Kube_Meta_Cache_TTL ${AZMON_KUBERNETES_METADATA_CACHE_TTL}
+#${KubernetesFilterEnabled}    Kube_Tag_Prefix     oms.container.log.la.var.log.containers.
+
```

Comment:
this has to be  **geneva.container.log.containers.**

Reply:
changed